public/index.html

```diff
@@ -347,13 +347,14 @@ <h3 class="text-2xl font-bold mb-2">Get notified for ticket sales & ruby updates
                 </article>
 
                 <article class="bg-slate-800 border-slate-700 border rounded-xl p-10 text-center shadow-lg">
-
-                    <img class="rounded-2xl overflow-clip mx-auto"
-                        src="https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExNnVjaXB2djdtYnJ4aXU4dDJkY3Q0dWg3dTR5bXNkdzV0emNzZmlrbCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/dRvEZLV0ORAmHT1L5u/giphy.gif"
-                        alt="">
-
-                    <p class="text-white mt-6">The conference of smiles | Made with love by <a class="text-red-400"
-                            href="https://bangkokrb.org/">bangkok.rb</a></p>
+                    <h3 class="text-2xl font-bold mb-2">The conference of smiles | Made with love by <a class="text-red-400" href="https://bangkokrb.org/#team">bangkok.rb</a></h3>
+                    <p class="text-slate-400 mb-6">Find us on the last Tuesday of every month here 👇</p>
+                    <div class="relative grid grid-cols-4">
+                        <img class="col-span-2 col-start-2 rounded-2xl overflow-clip mx-auto"
+                            src="/images/ruby_bangkok.png"
+                            alt="">
```

Comment:
I think if we bother to add the attribute, we might as well put "Bangkok.rb logo" in there.